scripts/run-migration.sh

```
@@ -0,0 +1,21 @@
+#!/bin/bash
+# Run database migration
+
+# Load environment variables
+source <(grep DATABASE_URL ../.env | xargs)
```

Shellcheck (0.11.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)